config/mpi_configs/mpi_sphere_aquaplanet_rhoe_equilmoist_clearsky.yml

```diff
@@ -4,7 +4,7 @@ moist: "equil"
 rad: "clearsky"
 precip_model: "0M"
 dt: "400secs"
-t_end: "4days"
+t_end: "400secs"
```

Comment:
I am assuming this will be rolled back. One way is to just run the GPU pipeline with this PR and compare with one of the older PRs!

Reply:
Yes, I committed this change by accident